server/src/models/Pokestop.js

```diff
@@ -596,25 +597,26 @@ module.exports = class Pokestop extends Model {
         if (onlyEventStops && eventStopPerms && displayTypes.length) {
           stops.orWhere((event) => {
             if (isMad && !hasMultiInvasions) {
-              event
-                .whereIn('incident_grunt_type', MADE_UP_MAD_INVASIONS)
-                .andWhere('incident_expiration', '>=', this.knex().fn.now())
+              event.where(function() { this.whereIn('incident_grunt_type', MADE_UP_MAD_INVASIONS).orWhere('character_display',0) })
```

Comment:
Let's change this to an arrow function to match the rest of the codebase.
```suggestion
              event.where((gruntType) => {
                gruntType.whereIn('incident_grunt_type', MADE_UP_MAD_INVASIONS)
                         .orWhere('character_display',0) 
              })
```